### PR TITLE
Added Language support for Hindi in language selection dropdown

### PR DESCRIPTION
**Pull request by agarwalprashant**
My First Pull Request to this Project !

1. Added a new option for Hindi Language in language selection dropdown in bottom right corner.

2. On selecting the hindi Option from the dropdown,the tooltip text for encryption shield icon (bottom left ) changes to hindi text.

Please suggest any changes . Thank you.

### CODE REVIEW

File: src/i18n.ts [hunk 1]
```diff
@@ -39,6 +39,7 @@ export const languages = [
     data: require("./locales/he-IL.json"),
     rtl: true,
   },
+  { lng: "hi-IN", label: "Hindi", data: require("./locales/hi-IN.json") },
```
Comment (lipis) [inline src/i18n.ts:42]: Could you write the label in Hindi?

File: src/locales/hi-IN.json [hunk 2]
```diff
@@ -170,7 +170,7 @@
     "zoomToFit": ""
   },
   "encrypted": {
-    "tooltip": ""
+    "tooltip": "आपके चित्र अंत-से-अंत एन्क्रिप्टेड हैं, इसलिए एक्सक्लूसिव्रॉव के सर्वर उन्हें कभी नहीं देखेंगे।"
```
Comment (lipis) [inline src/locales/hi-IN.json:173]: As for this.. the translation should happen here: https://crowdin.com/project/excalidraw/hi# and it's only at 7% right now.

File: src/i18n.ts (hunk 1 above)
Comment (agarwalprashant) [reply src/i18n.ts:42]: Sure @lipis, I will make this change

**Comment by lipis** [comment]
Hey! Could you update this PR? The translations are ready.

File: src/i18n.ts (hunk 1 above)
Comment (lipis) [reply src/i18n.ts:42]: हिन्दी

File: src/locales/hi-IN.json (hunk 2 above)
Comment (lipis) [reply src/locales/hi-IN.json:173]:
```suggestion
    "tooltip": ""
```

File: src/i18n.ts (hunk 1 above)
Comment (lipis) [reply src/i18n.ts:42]:
```suggestion
  { lng: "hi-IN", label: "हिन्दी", data: require("./locales/hi-IN.json") },
```

**Pull request merged**